src/providers/UserProvider.tsx

```diff
@@ -23,16 +24,25 @@ export default function UserProvider({ children, data }: UserProviderProps): JSX
         if (!data.user.timeZone) {
           // Should validate bowser time zone with supported time-zones returned by backend
           const browserTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
-          const updateResponse = await updateUserProfile({ ...data.user, timeZone: browserTimeZone });
-          if (updateResponse.requestSucceeded) {
-            data.reloadUser();
-            snackbar.pageSuccess(
-              strings.formatString(
-                strings.UPDATED_TIMEZONE_MSG,
-                <Link to={APP_PATHS.MY_ACCOUNT}>{strings.MY_ACCOUNT}</Link>
-              ),
-              strings.formatString(strings.UPDATED_TIMEZONE_TITLE, browserTimeZone)
+          const backendTimeZonesResponse = await getTimeZones();
```

Comment:
The localization provider will be fetching and providing this data.
Can we wait until that is implemented - we can move this check there later?

Reply:
We can get this in since you have already coded it up. No need to redo this.. we can just move the code later. Thoughts?

Reply:
Yes, let's move it later